index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="/style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+
+</head>
+
+<body>
+    <nav>
+        <img src="https://pmcvariety.files.wordpress.com/2017/09/youtube-logo.png?w=896" width="200" height="100" alt="" width="200" height="100">
+        <input class="searchbar" type="search">
+        <input class="searchbutton" type="button" value="search">
+        <div class="right">
+            <img src="https://via.placeholder.com/35x35/fff" alt="">
+            <img src="https://via.placeholder.com/35x35/000" alt="">
+            <img src="https://via.placeholder.com/35x35/fff000" alt="">
+        </div>
+    </nav>
+
+    <section id="content">
+        <div id="main_player">
+            <iframe width="98%" height="550" src="https://www.youtube.com/embed/ZnuwB35GYMY" frameborder="0"
+                allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+                allowfullscreen></iframe>
+            <h2>How To Embed A YouTube Video</h2>
+            <div class="meta_data">
+                <p id="views">12344 views</p>
+
+                <i class="material-icons c_icons">thumb_up</i>
+                <i class="material-icons  c_icons">thumb_down</i>
+                <i class="material-icons c_icons">share</i>
+            </div>
+
+            <div class="user">
+                <div id="avatar_container">
+                    <img id="avatar" src="https://i.imgur.com/MsUfB7u.png" height="50" width="50" alt="">
+                </div>
+
+                <div id="info">
+                    <h5>Current User</h5>
+                    <p>1111 subscirbers</p>
+                </div>
+
+                <div id="action">
+                    <img src="https://www.socialtalent.com/wp-content/uploads/blog-content/Subscribe_Button.png" width="150" height="50" alt="">
+                </div>
+            </div>
+        </div>
+
+        <div id="sugest">
+            <h4>Up next</h4>
+            <div class="sugesVideo" id="upNext">
+                <img src="https://via.placeholder.com/200x100" alt="">
+                <div class="sugesInfo">
+                    <h3>title</h3>
+                    <h4>11111 views</h4>
+                </div>
+            </div>
+            <div class="sugesVideo">
+                <img src="https://via.placeholder.com/200x100" alt="">
+                <div class="sugesInfo">
+                    <h3>title</h3>
+                    <h4>11111 views</h4>
+                </div>
+            </div>
```

Comment:
![sidebar](https://user-images.githubusercontent.com/31304943/69575099-e733db80-0fc9-11ea-9aab-3d26b07ffad0.png)

This sidebar should be on the right of the video as shown on the original page. You can use the `<aside>` tag for this.